webapp-django/crashstats/manage/jinja2/manage/site_overview.html

```diff
@@ -0,0 +1,69 @@
+{% extends "manage/base.html" %}
+
+{% block page_title %}{{ super() }} - Site Overview{% endblock %}
+
+{% block admin_title %}{{ super() }} - Site Overview{% endblock %}
+
+{% block mainbody %}
+<div class="panel">
+    <div class="body notitle">
+        <h3>Version info</h3>
+        {% for url, info in version_info.items() %}
+            <p>{{ url }}: {{ info }}</p>
+        {% endfor %}
+
+        <h3>Alembic status</h3>
+        <p>
+            {% if alembic_error %}
+                {{ alembic_error }}
+            {% else %}
+                HEAD: {{ alembic_version }}
+            {% endif %}
+        </p>
+
+        <h3>Django db status</h3>
+        {% if django_error %}
+            <p>{{ django_error }}</p>
+        {% else %}
+            <table class="data-table">
+                <tr>
+                    <th>id</th>
+                    <th>app</th>
+                    <th>name</th>
+                    <th>applied</th>
+                </tr>
+                {% for row in django_db_data %}
+                    <tr>
+                        {% for item in row %}
+                            <td>{{ item }}</td>
```

Comment:
Could we use attribute names instead of iterating through the rows? Right now the rows match up only because the HTML and view code are in sync, but defining that in one spot instead of two would avoid them being out of sync.